packages/web/docs/src/pages/self-hosting/get-started.mdx

```diff
@@ -63,7 +69,7 @@ After downloading the file, you need to set some environment variables.
 
 ```bash
 export DOCKER_REGISTRY="ghcr.io/kamilkisiela/graphql-hive/"
-export DOCKER_TAG=":latest"
+export DOCKER_TAG=":latest" # Pin this to an exact commit hash from our `main` branch
```

Comment:
Mention here that we publish a Docker image per each commit on `main`, and for PRs.